bootstrap.sh

```diff
@@ -1 +1,22 @@
 #!/bin/bash
+
+kubectl create namespace mysql
+kubectl apply -f .infrastructure/namespace.yml
+
+kubectl apply -f .infrastructure/statefulSet.yml
+
+echo "Checking the created MySQL resources..."
+kubectl get all -n mysql
+
+
+kubectl apply -f .infrastructure/configMap.yml
+kubectl apply -f .infrastructure/secret.yml
+kubectl apply -f .infrastructure/pv.yml
+kubectl apply -f .infrastructure/pvc.yml
+kubectl apply -f .infrastructure/clusterIp.yml
+kubectl apply -f .infrastructure/nodePort.yml
+kubectl apply -f .infrastructure/hpa.yml
+kubectl apply -f .infrastructure/deployment.yml
+
+echo "Checking the created todoapp resources..."
+kubectl get all -n todoapp
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_9_working_with_statefuset/assets/87720583/1e30b622-cd2e-4c49-b7c2-04c2b5af4931)